modules/serverless-creating-subscriptions.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * serverless/knative_eventing/serverless-subscriptions.adoc
+
+[id="serverless-creating-subscriptions_{context}"]
+= Creating a subscription
+
+You can create a subscription to connect a service or other event sink to a channel.
+
+[IMPORTANT]
+====
+Knative Eventing is a Technology Preview feature. The InMemoryChannel type is provided for development use only, and should not be used in a production environment.
+====
+
+.Prerequisites
+
+* You must have a current installation of xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[{ServerlessProductName}], including Knative Serving and Eventing, in your {product-title} cluster. This can be installed by a cluster administrator.
+* If you do not have an existing sink that you wish to use, you can create a Service to use as a sink by following the documentation on xref:../../serverless/serving-creating-managing-apps.adoc#serving-creating-managing-apps[Creating and managing serverless applications].
+* You will need a channel to connect your subscription to. See xref:../../serverless/knative_eventing/serverless-channels.adoc#serverless-channels[Using channels with Knative Eventing].
+
+.Procedure
+
+. Create a Subscription object to connect a channel to a service, by creating a YAML file containing the following:
++
+[source,yml]
+----
+apiVersion: messaging.knative.dev/v1beta1
+kind: Subscription
+metadata:
+  name: my-subscription <1>
+  namespace: default
+spec:
+  channel: <2>
+    apiVersion: messaging.knative.dev/v1beta1
+    kind: Channel
+    name: example-channel
+  delivery: <3>
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: error-handler
+  subscriber: <4>
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: example-service
```

Comment:
"Creating and managing serverless applications" page is showing `hello` ksvc. We also use `event-display` ksvc in some places. I didn't see `example-service` anywhere. Maybe better to use one of already existing examples?